src/hooks/useQuoteManager.js

```diff
@@ -0,0 +1,207 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+const AUTH_TOKEN = `bearer ${process.env.REACT_APP_API_TOKEN}`;
+
+export function useQuoteManager() {
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState();
+  const [searchResults, setSearchResults] = useState();
+  const [quoteResult, setQuoteResult] = useState();
+  const [applicationSuccess, setApplicationSuccess] = useState();
+
+  async function searchAddress(query) {
+    const {
+      address,
+      state,
```

Comment:
Do we need to leave this commented out code?

Reply:
The functionality may be added in the future but I think we can take it out for now and leave a comment that it may be updated in the future